NOTICE

```diff
@@ -0,0 +1,194 @@
+This content is produced and maintained by the Eclipse SmartHome project.
```

Comment:
What about the notice file in the root path -- there has been none for the current openHAB core repo.

Reply:
remove it

Reply:
The LICENSE file is enough there.

Reply:
I will remove it.